src/repel_boxes.cpp

```diff
@@ -939,6 +940,8 @@ DataFrame repel_boxes2(
 
   nanotime_t start_time = get_nanotime();
   nanotime_t elapsed_time = 0;
+  // convert to nanoseconds
+  max_time *= 1e9;
```

Comment:
unrelated to the current FR, but rather than convert to seconds each iteration 1000s of times, better to just do all the calculations in nanoseconds

Reply:
I think you are right, but I expect that the performance difference is negligible.

Reply:
Compiler might even optimize it away, but anyway i think it's a manageable small change